docs/source/using/experimental-features.adoc

```diff
@@ -87,3 +87,31 @@ PS> minishift.exe start `
 Be sure to specify a valid gateway and nameserver.
 Failing to do so will result in connectivity issues.
 ====
+
+
+[[set-fixed-ip]]
+== Set fixed IP Address
+
+While the VM gets assigned an IP address using DHCP, most hypervisors do not handle extending the lease time of the assignment.
+This can cause issues on restarts which assigns a new IP address to the instance. While the VM will start, it will conflict
+with the security certificates that have been generated before and the deployment will fail.
+
+For this purpose, we have added a functionality which can set a static address to the VM. This will prevent it from changing
+between restarts. However, it will not work on all the driver plugins at the moment due to the way the IP address is resolved.
+
+The following command will set the IP address that was assigned as fixed:
+
+----
+$ minishift ip --set-static 
+----
+
+If for some reason you prefer to use a dynamic assignment, you can use:
+
+----
+$ minishift ip --set-dhcp
+----
+
+[NOTE]
+====
+Currently not supported on KVM as the driver plugin relies on the DHCP offer to determine the IP address.
+====
```

Comment:
I think we should move this documentation from here and create a separate page similar to profile or add-on and in the documentation mention that this is experimental. Similar to SSHFS documentation. @minishift/minishift-dev comments?

Reply:
@LalatenduMohanty I think this makes perfect sense, but we can also address that in another issue/PR for more discussion.

Reply:
Since this covers a change in the organization, and maybe reconsidering how to handle 'experiment features' let's move this to a discussion on a new PR. several things related to network might have to be addressed.